src/assets/scripts/game/GameController.js

```diff
@@ -79,6 +80,7 @@ export default class GameController {
      */
     init_pre() {
         this.game_initializeBlurFunctions();
+        this.game_initializeStartTime();
```

Comment:
do you have eslint installed?  the linter should have thrown an error here with the underscore